README.md

```diff
@@ -36,11 +36,9 @@ git checkout master
 
 2. Set up the Engine development environment: See [the wiki page](https://github.com/flutter/flutter/wiki/Setting-up-the-Engine-development-environment).
 
-3. In **the engine repo**, add the prototype remote and switch to the prototype branch:
+3. In **the engine repo**, apply this playground's patches:
 ```
-git remote add dkwingsmt https://github.com/dkwingsmt/engine/
-git fetch dkwingsmt fffa6e19f5b71bedd55ac3a7fa6acfd85eda5983
-git checkout fffa6e19f5b71bedd55ac3a7fa6acfd85eda5983
+git apply /path/to/playground/patches/001-Add-multi-view-Flutter-Windows-C++-APIs.patch
 ```
 
 4. In **this repo**, edit `pubspec.yaml`, and add the following dependency override to use the custom `dart:ui` library:
```

Comment:
Also, while you're in here: Could you delete point 4? This is no longer necessary since the patch doesn't change any `dart:ui` API.

Reply:
Ah yup, good point!

Reply:
Removed